packages/build-tools/src/utils/artifacts.ts

```diff
@@ -19,7 +19,11 @@ export async function findArtifacts({
   /** If provided, will log error suggesting possible files to upload. */
   logger: bunyan | null;
 }): Promise<string[]> {
-  const files = await fg(patternOrPath, { cwd: rootDir, onlyFiles: false });
+  const files = path.isAbsolute(patternOrPath)
```

Comment:
Hmm, doesn't `fg` support absolute paths? That's what I thought…

Reply:
It failed to upload `/tmp/...` for me 🤔

Reply:
if that's macos, maybe you could try to use [temp-dir](https://www.npmjs.com/package/temp-dir) than `os.tmpdir()`